tools/configure/configure.ml

```diff
@@ -22,8 +22,8 @@ open CmdArgs.Prefs
 
 let (/) = Filename.concat
 
-let coq_version = "8.21+alpha"
-let vo_magic = 82099
+let coq_version = "9.0+alpha"
+let vo_magic = 90099
```

Comment:
Sorry, it looks like I had stopped my review half way.
Normally, the use of `99` is combined with the previous version number, so for 8.21, this was `82099`. Thus, I would expect `90099` to be the magic number for 9.1, and to keep the magic number unchanged in this PR.

Reply:
Indeed, good point